src/Adapter/Order/QueryHandler/GetOrderProductsForViewingHandler.php

```diff
@@ -179,9 +179,8 @@ public function handle(GetOrderProductsForViewing $query): OrderProductsForViewi
         $sorter = new Sorter();
         $products = $sorter->natural(
             $products,
-            $query->getProductsSorting()->getValue(),
-            'product_reference',
-            'product_supplier_reference'
+            $query->getProductsSorting()->getSortingOrder(),
```

Comment:
I am wondering if the way we did it in the past was a good idea or not. Maybe we should simply fetch the products, unsorted, and perform the sorting we want in the controller? This way it's a lot easier for anyone at the controller level to change the sorting.

Reply:
I see you never answered my comment 😅 hello @TheoAlloin ?

Reply:
oups, sorry for that, i forgot to answer here 😄 

Yes it seems a good way to do it. It will be really easier if we do it in this way. I'll try to provide a modification soon!